copy_java_files.sh

```diff
@@ -0,0 +1,11 @@
+# Use this to copy generated java files to hazelcast repo all at once. 
```

Comment:
generator.py already has an option where you can specify destination, why do you need this?

Reply:
yeah apparently it has, I didn't notice. I'll delete the script